sample_apps/run_example.sh

```diff
@@ -3078,6 +3194,52 @@ function run_test_simple_app_under_app_service() {
     run_simple_app_as_client_talk_to_Cert_Service
 }
 
+# ###########################################################################
+# Driver method to run simple_app steps with different pairs of public-key
+# and symmetric-key crypto algorithms. This interface is mainly a
+# testing interface to validate that different pairs of crypto algorithms
+# work correctly.
+# ###########################################################################
+function run_test-crypto_algorithms() {
+
+    run_cmd
+
+    start_certifier_service
+
+    subcase=0
+    # These valid-lists are found in certifier_algorithms.c
+    # public-key algo 'ecc-384' is failing ... Comment out for now ...
+    # for public_key_algo in rsa-3072 rsa-4096 ecc-384; do
+    for public_key_algo in rsa-3072 rsa-4096; do
+        # for symmetric_key_algo in aes-256-cbc-hmac-sha256; do
+        symmetric_key_algo="aes-256-cbc-hmac-sha256"
```

Comment:
Similarly, seems like we can only use this one named symmetric-key algorithm.

Otherwise, we trip-up in `src/certifier.cc`, here:

```
 901 bool certifier::framework::unprotect_blob(const string &enclave_type,
 902                                           int           size_protected_blob,
 903                                           byte *        protected_blob,
 904                                           key_message * key,
 905                                           int * size_of_unencrypted_data,
 906                                           byte *unencrypted_data) {
 907
[...]
 958   if (key->key_type() != Enc_method_aes_256_cbc_hmac_sha256) {
 959     printf("%s() error, line %d, unprotect_blob, unsupported encryption "
 960            "scheme: '%s'\n",
 961            __func__,
 962            __LINE__,
 963            key->key_type().c_str());
 964     return false;
 965   }
```

What is the support for other crypto algorithms listed as `private` in this file:

```
cc-sdb-vm:[9] $ grep Enc_ certifier_algorithms.cc |  grep -i private
const char * Enc_method_ecc_256_private           = "ecc-256-private";
const char * Enc_method_ecc_384_private           = "ecc-384-private";
const char * Enc_method_rsa_1024_private          = "rsa-1024-private";
const char * Enc_method_rsa_2048_private          = "rsa-2048-private";
const char * Enc_method_rsa_3072_private          = "rsa-3072-private";
const char * Enc_method_rsa_4096_private          = "rsa-4096-private";
```

We need to complete the list of symmetric-key algorithms supported to finish this testability matrix.